src/utils/get-sorted-nodes.ts

```diff
@@ -25,7 +25,10 @@ export const getSortedNodes = (
     nodes: ImportDeclaration[],
     order: PrettierOptions['importOrder'],
     importOrderSeparation: boolean,
+    importOrderCaseInsensitive: boolean,
```

Comment:
As this function receives four arguments now, it would be better to convert arguments to an object. 

```
export const getSortedNodes = ({
      nodes: ImportDeclaration[],
      order: PrettierOptions['importOrder'],
      importOrderSeparation: boolean,
      importOrderCaseInsensitive: boolean}
    ) => {}
```

Reply:
I can make this change next chance I get, it would also help fix the failing tests too (sorry, I thought they all passed before I pushed!)